api/AnimationEffectTiming.json

```diff
@@ -0,0 +1,54 @@
+{
+  "api": {
+    "AnimationEffectTiming": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEffectTiming",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "48",
+            "notes": "The Web Animations API is only enabled by default in Firefox Developer Edition and Nightly builds. You can enable it in beta and release builds by setting the preference <code>dom.animations-api.core.enabled</code> to <code>true</code>, and can disable it in any Firefox version by setting this preference to <code>false</code>."
```

Comment:
We use the "flags" directive to indicate that support (in release) is behind a flag. See the [docs](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#flags) for more info and an example of use.
In this case, we also have to reformulate the note to say that the preference is on by default in Firefox Developer Edition and Nightly builds.